### PR TITLE
nab salami

### PR DESCRIPTION
**Pull request by ghost**
fix gname ,nab

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Step 5                    | Change the data providers from JSON to MySQL in [thi
 /stats [player]              | Shows the player's stats along with advanced info, such as their ClientID and IP addresses | statspe.cmd.stats.advancedinfo
 /floatingstats add [name]    | Adds a floatingstat to your current person with a name specified by you                    | statspe.cmd.floatingstats
 /floatingstats list          | Lists all floatingstats on the server                                                      | statspe.cmd.floatingstats
-/floatingstats remove [name] | Removes a floatingstat                                                                     | statspe.cmd.floatingstats
+/floatingstats remove [name] | Removes a floatingstats                                                                     | statspe.cmd.floatingstats
```
Comment (SalmonDE) [inline README.md:31]: Pretty sure it's floatingstat

Comment (ghost) [reply README.md:31]: Wtf,is floatingstats  ,not floatingstat. @SalmonGER

Comment (ghost) [reply README.md:31]: '' s '' is not for plural . Is for stats.

Comment (SalmonDE) [reply README.md:31]: We talk about one single floatingstat there, but well let's merge it

**Pull request merged**